api/pom.xml

```diff
@@ -37,9 +36,9 @@
                 <version>9.40</version>
             </dependency>
             <dependency>
-                <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-bom</artifactId>
-                <version>${quarkus.version}</version>
+                <groupId>${quarkus.platform.group-id}</groupId>
```

Comment:
This is something that shows up in many current example Quarkus applications, and it's because there is a set of platforms [1] that could be selected based on the application's use case and possibly swapped. I may be missing something on that however - perhaps @metacosm may have a better understanding on the reason for using properties.

[1] https://github.com/quarkusio/quarkus-platform